src/components/BorrowerProfile/LendCta.vue

```diff
@@ -309,6 +308,7 @@
 								'md:tw-mb-0': !isSticky,
 								'md:tw-col-start-6 md:tw-col-span-7': !isSticky,
 								'md:tw-col-start-5 md:tw-col-span-6': isSticky,
+								'md:tw-hidden': isSticky,
```

Comment:
This element is currently hidden on medium resolutions. This happened with the addition of the jump links on medium resolutions and it is just hidden by the tw-z-1 under the medium sticky bar, with my changes, it popped out in front of the lend button, this just hides it explicitly when medium and sticky